### DISCUSSION TITLE
Ability to set a default appearance for public pages

### DISCUSSION THREAD

**Discussion by jenningsb2** [outline/outline#3275] [Ideas]
Hey team! 

I'd love the ability to set the default appearance mode for viewers of our documentation. I've gotten some feedback that some readers didn't like the dark mode, and they didn't realize it was configurable on their end.

To avoid this in the future, I'd love to be able to select light mode as the preferred default. If viewers would like to change to dark mode as their preference, fine, but it would be great to avoid starting there. 

<img width="484" alt="CleanShot 2022-03-21 at 11 05 55@2x" src="https://user-images.githubusercontent.com/29644050/159290808-0d69ed56-5e2b-4c23-909b-2abdf53aa7b3.png">

**Comment by tommoor** [comment]
> If viewers would like to change to dark mode as their preference, fine, but it would be great to avoid starting there.

The default is actually based on the users system settings, this is a pretty common pattern. We don't default to dark mode unless the users OS i set to dark mode.

**Reply by jenningsb2** [reply]
Got it! I saw that, but I was hoping to be able to override.

**Reply by jenningsb2** [reply]
Our brand colors aren't very dark-mode friendly, so all of our experiences are light-mode only at the moment.
